src/templates/dto/dto.template.ts

```diff
@@ -160,7 +160,9 @@ const getComponentGroups = (api: SwaggerApi) => {
 
         const key = isBodySuffix ? title.slice(0, -dtoBodySuffix.length) : title
```

Comment:
```suggestion
        const key = isBodySuffix ? title.slice(0, -1) : title
```

тут можно просто вот так написать

Reply:
done